atomic_reactor/utils/cachi2.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 Red Hat, Inc
+# All rights reserved.
+#
+# This software may be modified and distributed under the terms
+# of the BSD license. See the LICENSE file for details.
+
+"""
+Utils to help to integrate with cachi2 CLI tool
+"""
+
+from typing import Any, Dict
+
+
+def remote_source_to_cachi2(remote_source: Dict[str, Any]) -> Dict[str, Any]:
+    """Converts remote source into cachi2 expected params.
+
+    Remote sources were orignally designed for cachito. Cachi2 is not a direct
+    fork but has lot of similarities.
+    However, some parameters must be updated to be compatible with cachi2."""
+    cachi2_flags = sorted(
+        set(remote_source.get("flags", [])) - {"include-git-dir", "remove-unsafe-symlinks"}
+    )
+    cachi2_packages = []
+
+    for pkg_manager in remote_source["pkg_managers"]:
```

Comment:
Looking at the Cachito API docs, I realized that there's a `git-submodule` package manager (essentially just a flag that tells Cachito to clone submodules). We should omit that one, OSBS will need to be the one doing submodule cloning

Reply:
ok, so adding another feature to be backported to OSBS